plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -207,7 +207,7 @@ def __init__(self, addr: Optional[str] = None):
         if addr is None:
             addr = os.environ.get("OPEN_AEA_IPFS_ADDR", DEFAULT_IPFS_URL)
 
-        _, host, *_ = resolve_addr(addr)  # verify addr
+        _, host, *_ = resolve_addr(cast(str, addr))  # verify addr
```

Comment:
Again, `addr` could be None ? so add explicit raise if so?

Reply:
Addr can not be None, we make sure of that

https://github.com/valory-xyz/open-aea/blob/main/plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py#L200-L211